openhtf/io/frontend/__init__.py

```diff
@@ -170,9 +170,9 @@ def _discover(self):
           if (hostport, test_uid) not in self._watchers:
             self._watchers[hostport, test_uid] = TestWatcher(
                 hostport, remote_test,self._on_update_callback)
-      except (socket.error, station_api.StationUnreachableError) as e:
+      except station_api.StationUnreachableError:
         _LOG.debug('Station at %s is unreachable.', hostport)
-
+    
```

Comment:
lol, yeah i only just got my .vimrc re-setup properly.  we'll be doing a pylint pass over all this anyway.  these PRs are really fake-PRs that we're using just to work on a shared branch between us, so we're not putting much effort into review-ability, sorry :(